modules/EnsEMBL/Web/Component/Info/VariationGallery.pm

```diff
@@ -233,6 +233,7 @@ sub _get_pages {
     my $has_strains     = $variation_db->{'#STRAINS'} if $variation_db;
     my $has_LD          = ($variation_db && $variation_db->{'DEFAULT_LD_POP'}) ? 1 : 0;
     my $has_papers      = $object->count_citations ? 1 : 0; 
+    my $has_variation_source_db = $variation_db->{meta_info}->{1}->{'variation_source.database'}->[0];
```

Comment:
Can we call the new method you created named _has_variation_source_db_ from _modules/EnsEMBL/Web/Object/Variation.pm_  here instead? @at7